### PR TITLE
rest of maps

### PR DESCRIPTION
**Pull request by Maquiner**

### CODE REVIEW

File: xcell/mappers/mapper_eBOSSQSO.py [hunk 1]
```diff
@@ -21,82 +21,96 @@ def __init__(self, config):
         """
         self._get_defaults(config)
 
-        self.cat_data = []
-        self.cat_random = []
-        self.z_arr_dim = config.get('z_arr_dim', 50)
+        self.cats = {'data': [], 'random': []}
```
Comment (damonge) [inline xcell/mappers/mapper_eBOSSQSO.py:24]: As with decals, I'd make these `None` and `None` to be consistent with the other mappers.

File: xcell/mappers/mapper_eBOSSQSO.py [hunk 2]
```diff
@@ -21,82 +21,96 @@ def __init__(self, config):
...
         self.nl_coupled = None
         self.mask = None
 
+    def get_catalog(self, mod='data'):
+        if mod == 'data':
+            data_file = self.config['data_catalogs']
+        else:
+            data_file = self.config['random_catalogs']
+
+        if len(self.cats[mod]) == 0:
```
Comment (damonge) [inline xcell/mappers/mapper_eBOSSQSO.py:45]: And as before:
```
if self.cats[mod] == None:
    self.cats[mod] = []
```

**Review by damonge — CHANGES_REQUESTED** [review]
Do we want to save maps and masks? This is not done for DESY1gc or eBOSS

File: xcell/mappers/mapper_eBOSSQSO.py (hunk 2 above)
Comment (Maquiner) [reply xcell/mappers/mapper_eBOSSQSO.py:45]: I see! I was struggling with that. Thanks!

Comment (Maquiner) [reply xcell/mappers/mapper_eBOSSQSO.py]: resolved!

**Review by damonge — APPROVED** [review]
LGTM!

**Pull request merged**